client/state/data-layer/wpcom/sites/simple-payments/index.js

```diff
@@ -88,10 +90,15 @@ export function productToCustomPost( product ) {
 export function requestSimplePaymentsProduct( { dispatch }, action ) {
 	const { siteId, productId } = action;
 
-	dispatch( http( {
-		method: 'GET',
-		path: `/sites/${ siteId }/posts/${ productId }`,
-	}, action ) );
+	dispatch(
```

Comment:
Ran `prettier` on this file and it updated styling to match the rules. You can ignore changes below this line.